specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/apimgatewayConfigConnections.json

```diff
@@ -0,0 +1,350 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "delete": {
+        "tags": [
+          "ApiManagementGatewayConfigConnection"
+        ],
+        "operationId": "ApiManagementGatewayConfigConnection_Delete",
+        "description": "Deletes an existing API Management gateway config connection.",
+        "x-ms-examples": {
+          "ApiManagementGatewayDeleteGateway": {
+            "$ref": "./examples/ApiManagementDeleteGatewayConfigConnection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/GatewayNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ConfigConnectionIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/IfMatchRequiredParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The gateway config connection delete request was accepted.",
+            "headers": {
+              "location": {
+                "description": "Location header",
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "The gateway config connection does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "GatewayConfigConnectionBaseProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current provisioning state of the API Management gateway config connection ",
+          "enum": [
+            "Activating",
+            "Active",
+            "Created",
+            "Canceled",
+            "Failed",
+            "Stopped",
```

Comment:
>"Stopped", [](http://example.com/codeflow?start=12&length=10)

This is for SubscriptionLifecycle

For Stopped -> Succeeded

Reply:
Succeeded
Failed
Cancelled

Non-Terminal
- Creating
- Activating
- Updating
- Upgrading


Remove

Stopped,
Active